dist/emoji-en-US.json

```
@@ -8445,7 +8479,8 @@
     "health",
     "911",
     "hospital",
-    "vehicle"
+    "vehicle",
+    "hotfix"
```

```suggestion
    "vehicle"
```

Sorry I don't think this is general enough.